evalml/data_checks/datetime_format_data_check.py

```diff
@@ -36,14 +37,6 @@ def validate(self, X, y):
             >>> datetime_format_dc = DateTimeFormatDataCheck(datetime_column="dates")
```

Comment:
Do we need to update `.validate()`'s docstring a little bit more outside of the pydocstring?

Reply:
I can add another more descriptive line!